cli/dcoscli/data/help/package.txt

```diff
@@ -16,17 +21,37 @@ Usage:
 
 Options:
     --all                                  Apply the operation to all matching packages
+
     --app                                  Apply the operation only to the package's
-                                           application
+                                           marathon application
+
     --app-id=<app-id>                      The application id
+
     --cli                                  Apply the operation only to the package's CLI
+                                           command
+
+    --config                               Print the package's config.json, which contains
+                                           the configurable properties for the app
```

Comment:
I'd say `user configurable` instead of just `configurable` to make it obvious the user can change these.